sympy/functions/special/tests/test_bessel.py

```diff
@@ -33,6 +33,12 @@ def test_bessel_twoinputs():
         raises(TypeError, lambda: f(1))
         raises(TypeError, lambda: f(1, 2, 3))
 
+def test_bessel_series():
+    assert besselj(0, x).series(x) == 1 - x**2/4 + x**4/64 + O(x**6)
+    assert bessely(0, x).series(x) == (2*S.EulerGamma - 2*log(2) + 2*log(x) \
+        + x**2/2 + x**2*log(2)/2 - S.EulerGamma*x**2/2 - x**2*log(x)/2 \
+        + S.EulerGamma*x**4/32 - x**4*log(2)/32 - 3*x**4/64 + x**4*log(x)/32 + O(x**6))/pi
```

Comment:
The order term should not be in the parentheses as this is not an asymptotic expansion. Besides, it looks like it should be `O(x**6*log(x))` which dominates `O(x**6)`.

Reply:
Oh, right. For some reason, I had thought the `log(x)` was not needed, but that is incorrect.